pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java

```diff
@@ -135,6 +135,21 @@
     @Deprecated
     private boolean tlsEnabled = false;
 
+    @FieldContext(doc = "Name for the SSL provider")
+    private String sslProvider;
```

Comment:
this is called `tlsProvider` in other locations. Please rename.

Reply:
Done.